scripts/generate_ssl.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+openssl req -x509 -nodes -days 365 -subj "/C=CA/ST=QC/O=Company, Inc./CN=mydomain.com" -addext "subjectAltName=DNS:mydomain.com" -newkey rsa:2048 -keyout /etc/ssl/private/github.com.key -out /etc/ssl/certs/github.com.crt
+cp /etc/ssl/certs/github.com.crt /usr/local/share/ca-certificates/fake-github.com.crt
```

Comment:
You may need to run `update-ca-certificates` after this

Reply:
Added this in , nice call out!